resources/definitions/tronxy_base.def.json

```diff
@@ -0,0 +1,170 @@
+{
+    "name": "tronxy base printer",
+    "version": 2,
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": false,
+        "author": "Spanni26",
+        "manufacturer": "tronxy",
+        "file_formats": "text/x-gcode",
+        "platform": "ChiTu.stl", 
+        "platform_offset": [0, 0, 0],
+        "first_start_actions": ["MachineSettingsAction"],
+
+        "machine_extruder_trains": {
+            "0": "tronxy_base_extruder_0"
+        },
+
+        "has_materials": true,
+        "has_variants": true,
+        "has_machine_quality": true,
+        "variants_name": "Nozzle Size",
+
+        "preferred_variant_name": "0.4mm Nozzle",
+        "preferred_quality_type": "standard",
+        "preferred_material": "generic_pla"
+    },
+    "overrides": {
+        "machine_name": { "default_value": "Tronxy Base Printer" },
+        "machine_nozzle_size": { "default_value": 0.4},
+        "material_diameter": { "default_value": 1.75 },
+		"machine_start_gcode":{ "default_value":"G21\nG90\nM82\nM107 T0\nM140 S{material_bed_temperature}\nM104 S{material_print_temperature} T0\nM190 S{material_bed_temperature}\nM109 S{material_print_temperature} T0\nG28\nG92 E0\nG1 Z15.0 F{speed_travel}\nG0 E3 F200\nG92 E0\n"},
```

Comment:
Mixing up spaces and tabs here. We indent with spaces.

Reply:
Hmmm, strange. My VS-Code uses 4 spaces to indent. Saved it again....